liquibase/src/main/java/io/micronaut/liquibase/LiquibaseConfigurationProperties.java

```diff
@@ -87,28 +81,6 @@
 
     private boolean testRollbackOnUpdate = DEFAULT_TESTROLLBACKONUPDATE;
 
-    /**
-     * Ignores classpath prefix during changeset comparison.
-     * This is particularly useful if Liquibase is run in different ways.
-     *
-     * For instance, if Maven plugin is used to run changesets, as in:
-     * <code>
-     *      &lt;configuration&gt;
-     *          ...
-     *          &lt;changeLogFile&gt;path/to/changelog&lt;/changeLogFile&gt;
-     *      &lt;/configuration&gt;
-     * </code>
-     *
-     * {@link liquibase.Liquibase.Liquibase#listUnrunChangeSets(liquibase.Liquibase.Contexts, )} will
-     * always, by default, return changesets, regardless of their
-     * execution by Maven.
-     * Maven-executed changeset path name are not be prepended by
-     * "classpath:" whereas the ones parsed via AbstractLiquibase are.
-     *
-     * To avoid this issue, just set ignoreClasspathPrefix to true.
-     */
-    private boolean ignoreClasspathPrefix = DEFAULT_IGNORECLASSPATHPREFIX;
-
```

Comment:
I removed this because it's not available anymore on the Liquibase opensource edition.